tests/fixtures/migrations/0_base_01.sql

```diff
@@ -0,0 +1,110 @@
+-- items
+INSERT INTO swoop.input_item (item_uuid, item_id, collection) VALUES (
+  'f5db7f4d-7a72-441c-a9e5-ec2d88c66f5c',
+  'id1',
+  'collection1'
+);
+INSERT INTO swoop.input_item (item_uuid, item_id, collection) VALUES (
+  'a9b95ee3-3fee-4e02-8565-8137b2d036ed',
+  'id2',
+  'collection1'
+);
+INSERT INTO swoop.input_item (item_uuid, item_id, collection) VALUES (
+  '01887754-5a7c-430f-abc7-3b4ac0b1281d',
+  'id3',
+  NULL
+);
+INSERT INTO swoop.input_item (item_uuid, item_id, collection) VALUES (
+  'dc87a668-66ae-4ac6-86f1-afc8e467d9e7',
+  'id4',
+  NULL
+);
+
```

Comment:
I had deleted the SQL for the `input_item` and `item_payload` tables in my branch... why are we adding it back here?